judge/admin/comments.py

```diff
@@ -22,7 +22,7 @@ class CommentAdmin(VersionAdmin):
         (None, {'fields': ('author', 'page', 'parent', 'time', 'score', 'hidden')}),
         (_('Content'), {'fields': ('body',)}),
     )
-    list_display = ['author', 'linked_page', 'time']
+    list_display = ['author', 'linked_page', 'time', 'score', 'hidden']
```

Comment:
Is it possible to rename "Hide the comment" to "Hidden"?

Reply:
done, see the change in `Comment.hidden.verbose_name`